### PR TITLE
doc/ko: Copy English README.md to doc/ko directory

### PR DESCRIPTION
**Pull request by honggyukim**
This is a preparation for Korean translation for README.md.
The copied document is expected to be translated in Korean language.

Signed-off-by: Honggyu Kim <honggyu.kp@gmail.com>

### CODE REVIEW

**Comment by honggyukim** [comment]
I checked that there is no problem building other man pages even with this file so just copied it. Thanks.

File: doc/ko/README.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ Note that, there's an experimental support for dynamic tracing on x86_64
 which doesn't require such (re-)compilations.  Also recent compilers have
 some options to help uftrace to reduce tracing overhead with similar way
 (although it still needs recompilation of your program).  Please see
-[doc/uftrace-record.md](doc/uftrace-record.md) file.
+[uftrace-record.md](uftrace-record.md) file.
```
Comment (namhyung) [inline doc/ko/README.md]: I think it needs to keep original path in the square bracket.

Comment (honggyukim) [reply doc/ko/README.md]: I just fixed it. Thanks.

**Comment by honggyukim** [comment]
I rebased and updated it. Thanks. Please see the document below.
- https://github.com/honggyukim/uftrace/tree/doc/ko/readme/doc/ko

**Pull request merged**